cmd/db_migrate.go

```diff
@@ -9,21 +9,27 @@ import (
 	migrate "github.com/rubenv/sql-migrate"
 	"github.com/spf13/cobra"
 
+	"github.com/France-ioi/AlgoreaBackend/app/common"
 	"github.com/France-ioi/AlgoreaBackend/app/config"
 )
 
 func init() { // nolint:gochecknoinits
 
 	var migrateCmd = &cobra.Command{
-		Use:   "db-migrate [environment]",
```

Comment:
`./bin/AlgoreaBackend db-migrate dev` is much easier to type than `./bin/AlgoreaBackend db-migrate ALGOREA_ENV=dev`. Can we keep the command line arguments?